pycbc/conversions.py

```diff
@@ -626,6 +626,94 @@ def _det_tc(detector_name, ra, dec, tc, ref_frame='geocentric'):
 
 det_tc = numpy.vectorize(_det_tc)
 
+def _detector_to_optimal_orientation(detector_name, tc):
```

Comment:
Just to be consistent with names in the rest of the module, change `_detector_to_optimal_orientation` to `_optimal_orientation_from_detector`.